src/components/BasketballCourt/Border.tsx

```diff
@@ -6,11 +6,19 @@ import { changeTileColor } from "@/store/reducer/tileSlice";
 
 interface BorderProps {
   startPoint: ICourtStartPoint;
+  borderLength: number;
+  courtAreaXLength: number;
+  courtAreaYLength: number;
 }
-const Border: React.FC<BorderProps> = ({ startPoint }) => {
-  const { courtAreaXLength, courtAreaYLength, borderLength } = useStoreSelector(
-    (state) => state.courtSize
-  );
+const Border: React.FC<BorderProps> = ({
+  startPoint,
+  borderLength,
+  courtAreaXLength,
+  courtAreaYLength,
+}) => {
+  // const { courtAreaXLength, courtAreaYLength} = useStoreSelector(
+  //   (state) => state.courtSize
+  // );
```

Comment:
If these are needed please add them to TODO.

Reply:
Get it！thanks~